package/src/styles/animations.css

```
@@ -0,0 +1,25 @@
+@keyframes appearanceIn {
```

Stylelint (17.5.0)

[error] Expected keyframe name "appearanceIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)